platform/linux-dpdk/pktio/dpdk.c

```diff
@@ -664,34 +701,40 @@ static void stats_convert(struct rte_eth_stats *rte_stats,
 
 static int stats_pkt_dpdk(pktio_entry_t *pktio_entry, odp_pktio_stats_t *stats)
 {
+	const pktio_ops_dpdk_data_t *pkt_dpdk =
+		__retrieve_op_data(pktio_entry);
 	int ret;
 	struct rte_eth_stats rte_stats;
 
-	ret = rte_eth_stats_get(pktio_entry->s.pkt_dpdk.portid, &rte_stats);
+	ret = rte_eth_stats_get(pkt_dpdk->portid, &rte_stats);
 
 	if (ret == 0) {
 		stats_convert(&rte_stats, stats);
 		return 0;
-	} else {
-		if (ret > 0)
-			return -ret;
-		else
-			return ret;
 	}
+
+	if (ret > 0)
+		return -ret;
+	return ret;
 }
 
 static int stats_reset_pkt_dpdk(pktio_entry_t *pktio_entry)
 {
-	rte_eth_stats_reset(pktio_entry->s.pkt_dpdk.portid);
+	const pktio_ops_dpdk_data_t *pkt_dpdk =
+		__retrieve_op_data(pktio_entry);
+	rte_eth_stats_reset(pkt_dpdk->portid);
 	return 0;
 }
 
-const pktio_if_ops_t dpdk_pktio_ops = {
-	.name = "odp-dpdk",
+static pktio_ops_module_t dpdk_pktio_ops = {
+	.base = {
+		.name = "dpdk",
```

Comment:
Do not change interface names as part of this PR. Any name changes should be a separate PR where we can access application impact. In the case of DPDK devices this may be appropriate here, but not as part of this PR.

Reply:
Yes, will fix all these.